flipperdevices/flipperzero-firmware
# SubGhz: read and save static remotes. Create new static and dynamic remotes.

# What's new

 - SubGhz: add notifications subghz scene
 - SubGhz: add show all protocols in Read scene
 - SubGhz: add encode upload static protocol and keelog
 - SubGhz: add save/load static protocol
 - SubGhz: add upload static protocol and keelog
 - SubGhz: add create a remote (Pricenton, Nice Flo 12bit, Nice Flo 24bit, CAME 12bit, CAME 24bit, Gate TX, DoorHan)
 
# Verification 

- Compile and run
- Load a file with keys of dynamic protocols and examples of saved remotes to a USB flash drive
- Run the application and check the working ability

# Checklist (do not modify)
